sign/cosi/cosi.go

```diff
@@ -69,7 +69,7 @@ import (
 // CoSi struct. You can also give the full mask directly with SetMask().
 type CoSi struct {
 	// Suite used
-	suite crypto.Suite
+	group crypto.Group
```

Comment:
This answer holds as well as for the other places where I use a `Group`:
Using a `Group` is explicitely telling that the package is only using the `Point` & `Scalar` methods, and do not need a`Hash` or stg. You really want to add an additional level by writing `type Suite crypto.Group` ?